src/sticky.js

```diff
@@ -232,14 +232,26 @@ export default class Sticky extends Component {
     const scrollRect = getRect(scrollElement);
     const fixed = this.isFixed(holderRect, wrapperRect, boundaryRect, scrollRect);
 
+	function getClosestTransformedParent(el) {
```

Comment:
It's better to move this method to helpers, otherwise this function will be recreated every time